doc/man/8/ceph.rst

```diff
@@ -361,6 +361,37 @@ fs
 
 Manage cephfs file systems. It uses some additional subcommands.
 
+Subcommand ``add_data_pool`` adds an new data pool to the FS. Ths pool can
+be used for file layouts as an alternate location to store the file data.
+
+Usage::
+
+    ceph fs add_data_pool <fs-name> <pool name/id>
+
+Subcommand ``authorize`` creates a new client that will be authorized for the
+given path in ``<fs_name>``. Pass ``/`` to authorize for the entire FS.
```

Comment:
Pass ``/`` to `<path>` below ?

Reply:
Same questions as kotresh, also I think "for" can be removed from this line, isn't it?

Reply:
Passing `<path>` won't authorize client for entire FS.

Reply:
> Passing `<path>` won't authorize client for entire FS.

Wait, what? You mention the opposite in the doc, isn't it?